stable/grafana/templates/ingress.yaml

```diff
@@ -2,6 +2,7 @@
 {{- $releaseName := .Release.Name -}}
 {{- $servicePort := .Values.server.service.httpPort -}}
 {{- $path := .Values.server.ingress.path -}}
+{{- $root := . }}
```

Comment:
Honestly I'm not sure if this is best practice, but I'm not sure of another way to access to root template values from within the inside `range` scope.

Reply:
You can either do this or use `$` for the root context. I think the above is OK since it's consistent with the rest of the code.